### PR TITLE
feat: [DHIS2-11953] Quick actions-widget in enrollment dashboard

### PR DESCRIPTION
**Pull request by eirikhaugstulen**
https://jira.dhis2.org/browse/DHIS2-11953

Quick actions widget with navigation to "New event", "Make referral" and "Schedule an event".

### CODE REVIEW

File: src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentQuickActions/QuickActionButton/QuickActionButton.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+};
+
+type Props = {|
+    icon: Element<any>,
+    label: string,
+    onClickAction: () => void,
+    dataTest?: string,
+    disable?: ?boolean,
+    ...CssClasses,
+|}
```
Comment (JoakimSM) [inline src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentQuickActions/QuickActionButton/QuickActionButton.js:22]: Can we move this to a separate file? Also, I've found that it is, in most cases, better to use the annotation `disable?: boolean` to not allow null values; the benefit being that we can assign default values in the function parameters if we need it.

**Review by JoakimSM — COMMENTED** [review]
Looks good overall @EirikHaugstulen 👍 A couple of comments

**Comment by EirikHaugstulen** [comment]
Thanks @JoakimSM! 
Made some changes based on your review, give it a look when you have the time. ☺️

**Review by simonadomnisoru — APPROVED** [review]
LGTM!

**Review by JoakimSM — APPROVED** [review]

**Pull request merged**